lib/reporting/proofing_rate_report.rb

```diff
@@ -129,33 +129,31 @@ def reports
     # @return [Array<Float>]
     def blanket_proofing_rates(reports)
       reports.map do |report|
-        report.successfully_verified_users.to_f / report.idv_started
+        report.blanket_proofing_rates
       end
```

Comment:
We could make this:
```ruby
reports.map(&:blanket_proofing_rates)
```
and honestly probably just inline this (and the other two)